.github/workflows/fly.yml

```diff
@@ -0,0 +1,14 @@
+name: Fly Deploy
+on: [push]
+env:
+  FLY_API_TOKEN: ${{ secrets.FLY_API_TOKEN }}
+jobs:
+  deploy:
+    name: Deploy app
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v2
```

Comment:
v3?

Reply:
Fixed in 02034a8353e1ccf5f7ccd162c8dbf1dceeafba93